service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/SendDocumentTaskExecutor.java

```diff
@@ -73,7 +73,9 @@ public void execute(SendDocumentTaskDefinition taskDefinition) {
                 var messageId = randomIdGeneratorService.createNewId();
                 var filename = mainMessageId + "_" + i + MESSAGE_ATTACHMENT_EXTENSION;
                 var chunkPayload = generateChunkPayload(taskDefinition, messageId, filename);
-                var chunkedOutboundMessage = createChunkOutboundMessage(chunkPayload, chunk, taskDefinition.getDocumentContentType());
+
+                var chunkedOutboundMessage = createChunkOutboundMessage(chunkPayload, chunk, "application/octet-stream");
```

Comment:
Would it be better if the MIME type was a constant? As this is the pattern we've been using. See MESSAGE_ATTACHMENT_EXTENSION on line 74.

Reply:
Or the Tika static variable used in `DocumentToMHSTranslator.java`